.github/workflows/release.yml

```diff
@@ -131,9 +137,10 @@ jobs:
         env:
           RELEASE_NOTES: ${{ fromJSON(steps.get-release.outputs.result).body }}
           RELEASE_URL: ${{ fromJSON(steps.get-release.outputs.result).url }}
```

Comment:
Include a brief comment to explain the significance of setting the `commit_details` output.